spinn_utilities/log.py

```diff
@@ -245,10 +265,21 @@ def _atexit_handler(cls):
         messages = cls._repeat_log()
         if messages:
             level = logging.getLevelName(cls.__repeat_at_end)
-            print("\nThese log messages where generated at level {} or above"
-                  "".format(level), file=sys.stderr)
-            for message in messages:
-                print(message, file=sys.stderr)
+            if cls.__report_file:
+                if cls.__report_file:
```

Comment:
Something's wrong here, the same if statement is repeated twice...

Reply:
fixed